pkg/virt-launcher/virtwrap/device/sriov/hostdev.go

```diff
@@ -32,7 +35,11 @@ type pool interface {
 	Pop(key string) (value string, err error)
 }
 
-const AliasPrefix = "sriov-"
+const (
+	AliasPrefix = "sriov-"
+
+	MaxHostDevices = 32
```

Comment:
Correct me if I'm wrong. But even if the const is 32 and we have 100 sriov devices it would work. Isn't it? It is just an optimization.

Reply:
I think qemu/libvirt will not allow this as it will not be able to map the pci addresses on the guest bus.
But this value here, is not used to validate the schema, so your comment is correct if we ignore the low level implementation.

My claim is that we should add this validation on the schema, something we can add in a following PR.

Reply:
I'm saying `MaxHostDevices` is a confusing name. The const doesn't represent the max host devices, it is just internal number for optimizing the event channel. The event channel could have the length of 1 and it would still work.

Reply:
It may block the caller of the function from libvirt side, which I wanted to avoid here.
Ok, so how about this: `MaxCuncurrentHotPlugDevicesEvents` ?

Reply:
`MaxCuncurrentHotPlugDevicesEvents` is fine